addon/mixins/ajax-request.js

```diff
@@ -849,6 +847,8 @@ export default Mixin.create({
    * @return {Array} An array of JSON API-formatted error objects
    */
   normalizeErrorResponse(status, headers, payload) {
+    payload = isNone(payload) ? {} : payload;
```

Comment:
So, in `3.0.0` we're completely removing the implementation of `normalizeErrorResponse` in favor of just leaving it as a function that returns `payload`.  Given that, should we really be making changes to this method now?